ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnerGroups.vue

```diff
@@ -150,14 +150,19 @@ export default defineComponent({
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
+.pad-top {
+  padding-top: 0px;
+}
```

Comment:
You could get away with a vuetify helper here, `pt-0`

Reply:
nice... I need to get more familiar with those..

Reply:
For sure! They are quite helpful, especially when making those small adjustments.